relay-event-normalization/src/normalize/mod.rs

```diff
@@ -28,13 +28,7 @@ pub mod user_agent;
 pub mod utils;
 
 mod contexts;
-mod logentry;
-mod mechanism;
-mod processor;
 mod request;
-mod stacktrace;
```

Comment:
wondering why this mode where moved out of `normalize` module to top level?

Reply:
The `event` module uses `stacktrace::process_non_raw_frame` for stacktrace normalization. This results in 3 options on `stacktrace`:

1. Make it public in `normalize`. `normalize` is fully exported, so that'd result in `stacktrace` being unnecessarily exposed outside of the crate.
2. Make it `pub(crate)`. Workaround to option 1, but probably unnecessary as it's only used in `event`.
3. Make it a top-level module. I _think_ `normalize` will become redundant and we'll be able to merge it into the top-level module, so this seemed reasonable.

What do you think? Am I missing something?

Reply:
Do you feel strongly opinionated about it? Considering that the crate's purpose is to validate and normalize/process, what's the benefit of having a `normalize` submodule and what would go in or out of it?

Reply:
I'm fine with moving forward with this, just so we can deliver the changes faster. 
But I prefer better code organization in the future PRs.